sdk/samples/storage/blobs/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@ if(TRANSPORT_CURL)
   endif()
 endif()
 
+# Generate map file
+if(MSVC)
+    add_link_options(/MAP)
```

Comment:
For Windows, the optimizing flag that we should use is `/O1` since that gives the best code size. Right @barcharcraz?

https://docs.microsoft.com/en-us/cpp/build/reference/o-options-optimize-code?view=vs-2019